### DISCUSSION TITLE
PnP.Core.UrlUtility now missing in v1.3?

### DISCUSSION THREAD

**Discussion by CloudHotShot** [pnp/pnpframework#220] [General] [answered]
PnP.Core.UrlUtility class disappeared in latest release but I don't see a mention of this change. Did I miss?

**Comment by jansenbe** [comment, marked as answer]
@CloudHotShot : that's indeed correct, sorry it was left out from the change log. We try to keep all internal supporting classes internal in PnP Core SDK (in contrast with pnp framework where most are public due to historical reasons). If you need these it's best to copy the source code in your project (https://github.com/pnp/pnpcore/blob/dev/src/sdk/PnP.Core/Utilities/UrlUtility.cs)
